src/test/utils.ts

```diff
@@ -46,6 +46,9 @@ export function mockPrometheusServer() {
     .replyWithFile(200, __dirname + '/metadata.json')
     .get('/api/v1/series')
```

Comment:
I guess this, can be removed since it won't be possible to call the endpoint with the method GET